packages/config/tests/validate/fixtures/edge_functions_header_allowed/netlify.toml

```diff
@@ -0,0 +1,7 @@
+[[edge_functions]]
+function = "function2"
+path = "/function2"
+
+[edge_functions.header]
```

Comment:
toml interprets the values in an array as applied to the _previous_ item? In other words, these header declarations now apply to `function2`, but if we do

```
[[edge_functions]]
function = "function2"
path = "/function2"

[[edge_functions]]
function = "function3"
path = "/function3"

[edge_functions.header]
x-bar = "some-value"
x-foo = true
```

this would apply the headers to `function3`?

Reply:
Yes. 😑 

You can see that for yourself in https://pseitz.github.io/toml-to-json-online-converter/.